lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -323,6 +331,30 @@ type SignMetadata = "wallets"
     :> ReqBody '[JSON] ApiWalletSignData
     :> Post '[OctetStream] ByteString
 
+{-------------------------------------------------------------------------------
+                                  Assets
+
+  See also: https://input-output-hk.github.io/cardano-wallet/api/#tag/Assets
+-------------------------------------------------------------------------------}
+
+type Assets =
+    ListAssets
+    :<|> GetAsset
+
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/listAssets
+type ListAssets = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "assets"
+    :> Get '[JSON] [ApiAsset]
+
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/getAsset
+type GetAsset = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "assets"
+    :> Capture "policyId" (ApiT TokenPolicyId)
+    :> Capture "policyItem" (ApiT TokenName)
```

Comment:
If we're using the names in the specification, perhaps should this be "assetName"?

```suggestion
    :> Capture "assetName" (ApiT TokenName)
```

Reply:
Some previous drafts of the API spec used `policy_item` to refer to assets within a policy. I didn't yet update all the names in the code to use `asset_name`.